src/loggers.jl

```diff
@@ -131,15 +131,19 @@ Get the available log levels for a logger and their associated priorities.
 getlevels(logger::Logger) = logger.levels
 
 """
-    register(::Logger)
+    register(::Logger; force=false)
 
 Register an existing logger with Memento if it has not already been registered.
+
+NOTE: You can re-register a logger with `force=true`.
+This may be necessary if you want to explicitly register a parent logger after
+registering the child one, such as in submodule `__init__` functions.
```

Comment:
```suggestion
!!! note `force=true`
    You can re-register a logger with `force=true`.
    This may be necessary if you want to explicitly register a parent logger after
    registering the child one, such as in submodule `__init__` functions.
```

This makes a nice-looking admonition in the rendered docs (https://juliadocs.github.io/Documenter.jl/dev/showcase/#Admonitions)

Reply:
Cool, thanks!